integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -235,6 +251,8 @@ public int create(String path, @mode_t long mode, FuseFileInfo fi) {
         // Assuming I will never wrap around (2^64 open files are quite a lot anyway)
         mNextOpenFileId += 1;
       }
+      mFileSystem.setAttribute(uri,
```

Comment:
Can we store the following fields for FuseFileSystem to get the default user/group running this process:
```
    mFuseOwner = SecurityUtils.getOwnerFromLoginModule();
    mFuseGroup = SecurityUtils.getGroupFromLoginModule();
```
Then compare compare `userName` with `mFuseOwner` and `groupName` with `mFuseGroup` and only makes the RPC on string difference? In this way, we could avoid making extra RPCs in users/groups are the same as the default

Reply:
To derive similar values in master or 2.0, check https://github.com/Alluxio/alluxio/blob/1d50c373dcfa44ec4fa1b8bd05ab5cd49e72bd97/core/client/fs/src/main/java/alluxio/client/file/options/OutStreamOptions.java#L133